### PR TITLE
Add absolute dates

### LINKED ISSUE #5: Display absolute dates for deadlines somewhere
The BOD uses relative times (`120 days from...`), and it would be much more helpful to just have specific dates shown somewhere. 

I would definitely recommend updating the Checklist to show those (even if addition to the relative dates).

It would also be great to change text on the homepage, but if you don't want to change the text on the home page to keep it matched to the PDF, then I would link the `120 days from...` text to the Checklist, and then have the Checklist express it in absolute dates.

### PR DESCRIPTION
**Pull request by h-m-f-t**
This change adds actual, absolute dates to the /guide page (fixes #5) whereby agencies must comply with BOD 18-01. It also:
* adds links from the index page (the text of the BOD) to /guide
* fixes an error that incorrectly put initial DMARC compliance in the 120 day bucket instead of the 90 day bucket where it belongs
* minor style changes to the index page

### CODE REVIEW

File: pages/guide.md [hunk 1]
```diff
@@ -13,17 +13,18 @@ This page provides implementation guidance for [Binding Operational Directive 18
 
 ### Checklist
 
-* Within **30 days** of BOD issuance, submit an "Agency Plan of Action" to <FNR.BOD@hq.dhs.gov> and begin implementing the plan.
-* At **60 days** after BOD issuance (**and at every 30 days until full implementation**), provide a status report of plan implementation to <FNR.BOD@hq.dhs.gov>.
-* Within **90 days** of BOD issuance, configure all internet-facing mail servers to offer STARTTLS.
-* By **120 days** after BOD issuance:
+* Within **30 days** of BOD issuance *(November 15th, 2017)*, submit an "Agency Plan of Action" to <FNR.BOD@hq.dhs.gov> and begin implementing the plan.
+* At **60 days** *(December 15th, 2017)* after BOD issuance (**and at every 30 days until full implementation**), provide a status report of plan implementation to <FNR.BOD@hq.dhs.gov>.
+* Within **90 days** *(January 15, 2018)* of BOD issuance:
```
Comment (IanLee1521) [inline pages/guide.md]: Oct 16 + 90 days is actually January 14, not 15 (though I imagine this was shifted due to the 14th being a Sunday).

Comment (h-m-f-t) [reply pages/guide.md]: Yeah, that date was shifted.

Comment (IanLee1521) [reply pages/guide.md]: Nod nod. Does that need to be called out explicitly do you think, or that is near enough and expected enough that would happen?

Comment (h-m-f-t) [reply pages/guide.md]: Sorry, missed your comment before merging. 

The Nov. 15th (16th) date was already communicated out to agency security leads directly, and it's also _this week_. Also, not many know yet that these dates are on the sites; the chance of confusion seems small.🤞

**Pull request merged**